omega/omega-transaction/src/test/java/org/apache/servicecomb/pack/omega/transaction/ForwardRecoveryTest.java

```diff
@@ -155,7 +155,7 @@ public void throwExceptionWhenRetryReachesMaximum() throws Throwable {
           containsString("Failed to handle tx because it is interrupted")));
     }
 
-    assertThat(messages.size(), is(3));
+    await().atMost(10, SECONDS).until(() -> messages.size() == 3);
```

Comment:
Please don't use the lamda founction in omega side which only supports  Java7 syntax.

Reply:
done